TACC/Frontera-Portal
# task/FP-789-add shared workspace modal

## Overview: ##

Add shared workspace modal

## PR Status: ##


## Related Jira tickets: ##

* [FP-213](https://jira.tacc.utexas.edu/browse/FP-213)
* [FP-789](https://jira.tacc.utexas.edu/browse/FP-789)


## Summary of Changes: ##

- "Add shared workspace" modal
- Project member list component

## Testing Steps: ##
1. Create a project using your own account
2. Using an incognito window, log in with a test account and try to create a project, granting your normal account read access using the Add button
3. Make sure both accounts are able to read any project they have access to without having to push keys

## UI Photos:

![image](https://user-images.githubusercontent.com/17181582/101522533-fd9aa400-394c-11eb-9034-08bca51b3a22.png)

## Notes: ##

- The Add member component needs to be refactored out and reconciled with similar functionality in the Allocations manager
- The Add member component does not clear the selected member after adding. Does anyone have a suggestion on the best way to clear the Input element programmatically?
